skopt/optimizer/optimizer.py

```diff
@@ -166,6 +168,14 @@ def _check_arguments(self, base_estimator, n_random_starts, acq_optimizer):
             warnings.warn("The 'auto' option for the acq_optimizer will be "
                           "removed in 0.4.")
             acq_optimizer = "lbfgs"
+
+        if not acq_optimizer == "sampling" and \
```

Comment:
@MechCoder That makes a lot of sense, thanks! I've addressed that on the next commit.